src/components/LocationsList.js

```diff
@@ -0,0 +1,89 @@
+// @flow
```

Comment:
let it be just locationsList,TapsList etc..Don't want to implement generic DAOEntityList until I notice common things in those.